### ISSUE TITLE
Cluster exploring giving websocket connection errors continuously

### ISSUE THREAD

**Issue by rjbaat** [rancher/rancher#38931]
I have websocket connection issue notifications popping up on 2.6.7. There are multiple users having the same issue on multiple versions of Rancher. Like 2.6.8. See issue: https://github.com/rancher/rancher/issues/32934#issuecomment-1246944875

I added this issue to generate extra attention for the problem that occurs, since the other issue is already closed. 

The message is: 

```
The connection to wss://my.rancher.host/v1/subscribe closed unexpectedly Thu, Aug 25 2022 11:17:06 am. Retrying...
Websocket Disconnected
The connection to wss://my.rancher.host/v1/subscribe closed unexpectedly 
```

And in the logs it tels me: 

```
Error during subscribe websocket: close sent
Error during subscribe write tcp 172.17.0.3:443->xxx.xxx.xxx.xxx:52499: write: broken pipe
```

Can you please take time to address this issue? It is quite blocking, since the interface is really unusable when having all the notifications popping up continuously.

**Comment by konstantin-921** [comment]
We are also facing this issue on 2.6.8.
We are using 4 installations of rancher + downstream cluster. We got this problem in all of our clusters (including test installation without any workload at all). That is, there were different conditions everywhere: resources, network, etc.

Also I can see following error in rancher pods:
`[ERROR] Unknown error: Operation cannot be fulfilled on preferences.management.cattle.io "cluster": the object has been modified; please apply your changes to the latest version and try again`

I used even the fresh Mozilla installation instead of Chrome. The same problem.

**Comment by rak-phillip** [comment]
@rjbaat thanks for raising this issue and helping to make Rancher better!

@gaktive this might be related to the open dashboard issue rancher/dashboard#6785. What do you think?

**Comment by gaktive** [comment]
Yep. As @rak-phillip noted, https://github.com/rancher/dashboard/issues/6785 is where we're working to address this issue, so this ticket is a duplicate. There will be further investigations into the websocket behaviour later on but we aim to minimize the alerts since this makes the dashboard noisy.

If we can't get that into 2.6.9, we do have a temporary measure in place with an option to disable the alerts: https://github.com/rancher/dashboard/issues/6793
